front/lib/api/files/upsert.ts

```diff
@@ -404,18 +424,23 @@ export async function processAndUpsertToDataSource(
     });
   }
 
-  const r = await getConversation(auth, file.useCaseMetadata.conversationId);
+  const workspace = auth.getNonNullableWorkspace();
 
-  if (r.isErr()) {
+  const conversation = await Conversation.findOne({
```

Comment:
We should not have findOne outside of resources... here we don't have resources but we do have `getConversationWithoutContent`